packages/p2p/src/pages/my-ads/create-ad/create-ad.tsx

```diff
@@ -1,22 +1,22 @@
-import * as React from 'react';
-import { Loading } from '@deriv/components';
-import { observer } from 'mobx-react-lite';
+import React from 'react';
+import { observer } from '@deriv/stores';
 import { localize } from 'Components/i18next';
 import PageReturn from 'Components/page-return';
-import { useStores } from 'Stores';
-import CreateAdForm from './create-ad-form.jsx';
+import { useStores } from 'Stores/index';
+import CreateAdForm from './create-ad-form';
 
 const CreateAd = () => {
     const { my_ads_store } = useStores();
+    const { setApiErrorMessage, setShowAdForm } = my_ads_store;
 
     const onClickBack = () => {
-        my_ads_store.setApiErrorMessage('');
-        my_ads_store.setShowAdForm(false);
+        setApiErrorMessage('');
+        setShowAdForm(false);
     };
     return (
         <React.Fragment>
             <PageReturn onClick={onClickBack} page_title={localize('Create new ad')} />
-            {my_ads_store.is_form_loading ? <Loading is_fullscreen={false} /> : <CreateAdForm />}
```

Comment:
why did we remove the loader?

Reply:
this loader is used only for edit form case, where we wait for the values to be prefilled. is_form_loading value is always false for createad case